codemodder/codemods/transformations/clean_imports.py

```diff
@@ -247,22 +371,22 @@ def _triage_imports(all_imports, src_path):
 
     config = Config(src_paths=(src_path,))
 
-    for imp in all_imports:
+    for stmt in all_imports:
+        imp = stmt.body[0]
         section = isort.place.module(_get_name(imp), config)
 
         if section == isort.sections.FUTURE:
-            future.append(imp)
+            future.append(stmt)
         elif section == isort.sections.STDLIB:
-            stdlib.append(imp)
+            stdlib.append(stmt)
         elif section == isort.sections.LOCALFOLDER:
-            local.append(imp)
+            local.append(stmt)
         elif section == isort.sections.FIRSTPARTY:
-            first_party.append(imp)
+            first_party.append(stmt)
         else:
-            third_party.append(imp)
+            third_party.append(stmt)
 
-    imports_by_section = [future, stdlib, third_party, first_party, local]
-    return imports_by_section
+    return [future, stdlib, third_party, first_party, local]
```

Comment:
Not a huge dea but it feels like we should really be returning a `dict` or custom class of some type here.

Reply:
Good call, it was done this way because the order matters (it's isort's default one). But a dict could make it more modular.